packages/webpack-config-single-spa/lib/webpack-config-single-spa.js

```diff
@@ -94,18 +97,6 @@ function webpackConfigSingleSpa(opts) {
       new BundleAnalyzerPlugin({
         analyzerMode: webpackConfigEnv.analyze ? "server" : "disabled",
       }),
-      new UnusedFilesWebpackPlugin({
```

Comment:
This plugin doesn't work with webpack 5 (rip). See https://github.com/tomchentw/unused-files-webpack-plugin/issues/39